### ISSUE TITLE
pip install'ing v4.0.2 doesn't bring the compat directory

### ISSUE THREAD

**Issue by OdyX** [SethMMorton/natsort#30]
natsort v4.0.2 installed with pip doesn't come with the compat directory, which then breaks running natsort:

```
$ mkdir natsort-test; virtualenv natsort-test/; natsort-test/bin/pip install natsort==4.0.2 ; ls -lah natsort-test/lib/python2.7/site-packages/natsort/
Running virtualenv with interpreter /usr/bin/python2
New python executable in natsort-test/bin/python2
Also creating executable in natsort-test/bin/python
Installing setuptools, pip...done.
Downloading/unpacking natsort==4.0.2
  Downloading natsort-4.0.2-py2.py3-none-any.whl
Installing collected packages: natsort
Successfully installed natsort
Cleaning up...
total 160K
drwxr-xr-x  2 diidier diidier  360 jun 25 14:49 .
drwxr-xr-x 10 diidier diidier  240 jun 25 14:49 ..
-rw-r--r--  1 diidier diidier  772 jun 25 14:49 __init__.py
-rw-r--r--  1 diidier diidier 1.1K jun 25 14:49 __init__.pyc
-rw-r--r--  1 diidier diidier 2.4K jun 25 14:49 locale_help.py
-rw-r--r--  1 diidier diidier 2.5K jun 25 14:49 locale_help.pyc
-rw-r--r--  1 diidier diidier 8.2K jun 25 14:49 __main__.py
-rw-r--r--  1 diidier diidier 8.0K jun 25 14:49 __main__.pyc
-rw-r--r--  1 diidier diidier  21K jun 25 14:49 natsort.py
-rw-r--r--  1 diidier diidier  22K jun 25 14:49 natsort.pyc
-rw-r--r--  1 diidier diidier 7.3K jun 25 14:49 ns_enum.py
-rw-r--r--  1 diidier diidier 7.5K jun 25 14:49 ns_enum.pyc
-rw-r--r--  1 diidier diidier  12K jun 25 14:49 unicode_numbers.py
-rw-r--r--  1 diidier diidier  11K jun 25 14:49 unicode_numbers.pyc
-rw-r--r--  1 diidier diidier  15K jun 25 14:49 utils.py
-rw-r--r--  1 diidier diidier  11K jun 25 14:49 utils.pyc
-rw-r--r--  1 diidier diidier  150 jun 25 14:49 _version.py
-rw-r--r--  1 diidier diidier  322 jun 25 14:49 _version.pyc

$ ./natsort-test/bin/natsort
Traceback (most recent call last):
  File "./natsort-test/bin/natsort", line 7, in <module>
    from natsort.__main__ import main
  File "/tmp/natsort-test/local/lib/python2.7/site-packages/natsort/__init__.py", line 10, in <module>
    from natsort.natsort import (
  File "/tmp/natsort-test/local/lib/python2.7/site-packages/natsort/natsort.py", line 28, in <module>
    from natsort.compat.py23 import u_format
ImportError: No module named compat.py23
```

**Comment by maturano** [comment]
Same problem here U_U

**Comment by mikeage** [comment]
It also breaks https://github.com/FalconSocial/pre-commit-python-sorter, which is a fairly popular pre-commit hook (normally I hate the +1 comments, but...)

**Comment by rm--** [comment]
I am not sure if this is the correct place. But perhabs it would be a good idea to add a tox env for [check-manifest](https://github.com/mgedmin/check-manifest) to check the files before pushing to PYPI. We used it in our company since some months and we are very happy with it.

**Comment by SethMMorton** [comment]
Oh man, sorry guys!!!!!!!!!

I have just fixed this and released version 4.0.3, and have validated that this installs correctly.

**Comment by mikeage** [comment]
Yep, working great. Thanks so much!

**Comment by SethMMorton** [comment]
OH THANK GOD!  Thanks for letting me know.

I will definitely update my release procedure to avoid boo-boos like this.

I 'm going to reopen and keep this open for a week or so in case someone updated in that 8 hour window but didn't run right away... then this issue will be easy to find.

For reference, the update came in commit 03cc17b9f5c9e6ff30f5c12ed276e672c91b0371 (tag v4.0.3), and is fixed in natsort version 4.0.3 on PyPI.
